shared/js/content-scripts/onboarding.js

```diff
@@ -0,0 +1,15 @@
+(function () {
+    // Note: this is injected by atb.es6.js
+    // This is currently used for the cross product promotion on desktop experiment.
```

Comment:
I think we can simplify this to say it is only used on duckduckgo.com to assist with user onboarding.

Reply:
https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/452/commits/9819655facb88e89a8382a138a8618a040e9a4df